tests/api/test_recents.py

```diff
@@ -0,0 +1,153 @@
+# SPDX-License-Identifier: GPL-3.0+
+
+from __future__ import unicode_literals
+import json
+from datetime import datetime
+
+import pytest
+
+from estuary.models.koji import KojiBuild
+from estuary.models.bugzilla import BugzillaBug
+from estuary.models.distgit import DistGitCommit
+from estuary.models.errata import Advisory
+from estuary.models.freshmaker import FreshmakerEvent
+
+
+@pytest.mark.parametrize('expected', [
```

Comment:
Since you're only doing one test, `parametrize` isn't needed. Can you refactor the test to not use it?